libs/context/application/src/impl/analyze_project_handler_impl.py

```
@@ -55,10 +77,60 @@ def _build_entity(self, command: AnalyzeProjectCommand) -> AnalysisJob:
+                target_id=job.target_id,
+                status="completed",
+                result={
+                    "technical_stack": project_data.technical_stack,
+                    "semantic_context": project_data.semantic_context,
+                    "architectural_context": project_data.architectural_context,
+                },
+                started_at=job.started_at,
+                finished_at=datetime.now(timezone.utc),
+                error="",
+            )
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)